cli/connect/internal/luabody/eval_func_body.lua

```diff
@@ -23,7 +23,7 @@ if not fun then
     fun, errmsg = loadstring(cmd)
 end
 if not fun then
-    return yaml.encode({box.NULL})
+    return box.error(box.error.PROC_LUA, errmsg)
```

Comment:
Your solution causes the connection closing in case of invalid command. I think it would be better just to print the error:
```suggestion
    return yaml.encode({ errmsg })
```

Reply:
For some reason I didn't try to just `return {error = errmsg}`

it save same behavior
```
echo "hello)" | tt connect --evaler '_, errmsg = loadstring(cmd); return { error = errmsg }' $TARANTOOL_CONSOLE_URI
---
- error: '[string "hello)"]:1: ''='' expected near '')'''
...
```

unlike `return yaml.encode({ error = errmsg })`
```
echo "hello)" | tt connect --evaler '_, errmsg = loadstring(cmd); return yaml.encode({ error = errmsg })' $TARANTOOL_CONSOLE_URI
---
- |
  ---
  error: '[string "hello)"]:1: ''='' expected near '')'''
  ...
...
```